NEWS.md

```diff
@@ -1,3 +1,11 @@
+# ospsuite.utils 1.2.0.9000
+
+MAJOR CHANGES
+
+* Cloning method is now disabled for `Printable` R6 class. This entails that all
+  subclasses that inherit from this class will not be able to use cloning,
+  **even if** they explicitly set `cloneable = TRUE`.
```

Comment:
@IndrajeetPatil Did you actually **test** this? This is not the case!

Without overriding `cloneable`:

```
DataSet <- R6::R6Class(
  "DataSet",
  inherit = DotNetWrapper,
#  cloneable = FALSE,
  active = list(
```

```
> ds1 <- DataSet$new("myDs")
> ds1$clone()
DataSet: 
   Name: myDs 
   X dimension: Time 
   X unit: h 
   Y dimension: Concentration (mass) 
   Y unit: mg/l 
   Error type: 
   Error unit: 
   Molecular weight: 
   LLOQ: 
   Meta data: 
list()
```

With overriding:

```
DataSet <- R6::R6Class(
  "DataSet",
  inherit = DotNetWrapper,
  cloneable = FALSE,
  active = list(
```

```
> ds1 <- DataSet$new("myDs")
> ds1$clone()
Error: attempt to apply non-function
```


This documentation must be changed.